README.md

```diff
@@ -244,6 +252,60 @@ This is useful on development machines. It allows you to:
 
 All while running Vaultenv without any CLI args.
 
+It can happen that conflicting configuration values are send to Vaultenv. An example
+would be a global secret file, which is overwritten by a project specific configuration file.
+The order of precedent (from least important to most important) is as follows:
+ - `/etc/vaultenv.conf`
+ - `$HOME/.config/vaultenv/vaultenv.conf`
+ - `$CWD/.env`
+ - environment variables
+ - command line options
+
+This means that any command line option that is present would overwrite any other configuration.
+If an option is not specified, the default is used. The defaults are as follows:
+```
+Host:           localhost
```

Comment:
It'd be a bit more user friendly if these were the actual strings that the user needs to enter in the config file / on the CLI instead of the pretty printed names.